admin/proc-rhdh-monitoring-logging-aws.adoc

```diff
@@ -0,0 +1,149 @@
+
+After ingesting Prometheus metrics to the created workspace, you can configure the metrics scraping to extract data from pods based on specific pod annotations.
+
+=== Configuring annotations for monitoring
+
+You can configure the annotations for monitoring in both Helm deployment and Operator-backed deployment.
+
+Helm deployment::
++
+--
+To annotate the backstage pod for monitoring, update your `values.yaml` file as follows:
+
+[source,yaml]
+----
+upstream:
+  backstage:
+    # --- TRUNCATED ---
+    podAnnotations:
+      prometheus.io/scrape: 'true'
+      prometheus.io/path: '/metrics'
+      prometheus.io/port: '7007'
+      prometheus.io/scheme: 'http'
+----
+--
+
+Operator-backed deployment::
++
+--
+.Procedure
+
+. As an administrator of the operator, edit the default configuration to add Prometheus annotations as follows:
++
+[source,bash]
+----
+# Update OPERATOR_NS accordingly
+$ OPERATOR_NS=rhdh-operator
+$ kubectl edit configmap backstage-default-config -n "${OPERATOR_NS}"
+----
+
+. Find the `deployment.yaml` key in the ConfigMap and add the annotations to the `spec.template.metadata.annotations` field as follows:
++
+[source,yaml]
+----
+deployment.yaml: |-
+  apiVersion: apps/v1
+  kind: Deployment
+  # --- truncated ---
+  spec:
+    template:
+      # --- truncated ---
+      metadata:
+        labels:
+         rhdh.redhat.com/app:  # placeholder for 'backstage-<cr-name>'
+        # --- truncated ---
+        annotations:
+          prometheus.io/scrape: 'true'
+          prometheus.io/path: '/metrics'
+          prometheus.io/port: '7007'
+          prometheus.io/scheme: 'http'
+  # --- truncated ---
+----
+
+. Save your changes.
+--
+
+.Verification
+
+To verify if the scraping works:
+
+. Use `kubectl` to port-forward the Prometheus console to your local machine as follows:
++
+[source,bash]
+----
+$ kubectl --namespace=prometheus port-forward deploy/prometheus-server 9090
+----
+
+. Open your web browser and navigate to `http://localhost:9090` to access the Prometheus console.
+. Monitor relevant metrics such as `process_cpu_user_seconds_total`.
+
+== Logging with Amazon CloudWatch logs
+
+Logging within the {product} relies on the https://github.com/winstonjs/winston[winston library]. By default, logs of the debug level are not recorded. To activate debug logs, you must set the environment variable `LOG_LEVEL` to debug in your {product} instance.
+
+=== Configuring the application log level
+
+You can configure the application log level in both Helm deployment and Operator-backed deployment.
+
+Helm deployment::
++
+--
+To update the logging level, add the environment variable `LOG_LEVEL` in your Helm chart's `values.yaml` file:
```

Comment:
I think it should be "in". We're making changes **in** the file.

Reply:
@hmanwani-rh Both of them ("in" and "to") are grammatically correct. The use of "to" seems more appropriate here to me. Because, it directly states where the environment variable should be added in the `values.yaml` file. I believe, this makes the instruction more clearer. What do you think?

Reply:
Either option works for me. If you're insistent on it, I'll go ahead and make the change.